Spigot-Server-Patches/0539-Fix-anvil-not-working-when-level-req-is-0.patch

```diff
@@ -0,0 +1,47 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Tuck <jan@tuck.dk>
+Date: Mon, 6 Jul 2020 02:56:45 +0200
+Subject: [PATCH] Fix anvil not working when level req is 0
+
+
+diff --git a/src/main/java/net/minecraft/server/ContainerAnvil.java b/src/main/java/net/minecraft/server/ContainerAnvil.java
+index fc2038df89f25c07f6f853f6df41fe9b203c3585..9c32c9db9d70d7d033445c76319477ca509ffe58 100644
+--- a/src/main/java/net/minecraft/server/ContainerAnvil.java
++++ b/src/main/java/net/minecraft/server/ContainerAnvil.java
+@@ -38,7 +38,8 @@ public class ContainerAnvil extends ContainerAnvilAbstract {
+ 
+     @Override
+     protected boolean b(EntityHuman entityhuman, boolean flag) {
+-        return (entityhuman.abilities.canInstantlyBuild || entityhuman.expLevel >= this.levelCost.get()) && this.levelCost.get() > 0;
++        // return (entityhuman.abilities.canInstantlyBuild || entityhuman.expLevel >= this.levelCost.get()) && this.levelCost.get() > 0; //Paper - Changed
```

Comment:
Just modify the thing instead of commenting it out, you just create more diff noise

Reply:
Oh my bad